### PR TITLE
Better logging of high level tabulation activities, especially during sequential race.

### PR DESCRIPTION
**Pull request by moldover**
Also removed some redundant info.    Helps with #317

### CODE REVIEW

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 1]
```diff
@@ -105,11 +107,28 @@ void tabulate() throws TabulationCancelledException {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     if (config != null && config.validate() && setUpLogging(config)) {
+      try {
+        Logger.log(Level.INFO, "Begin config file contents:");
+        BufferedReader reader = new BufferedReader(new FileReader(configPath));
+        String line = reader.readLine();
+        while(line != null) {
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: missing space

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 2]
```diff
@@ -105,11 +107,28 @@ void tabulate() throws TabulationCancelledException {
     Logger.log(Level.INFO, "Starting tabulation session...");
     ContestConfig config = ContestConfig.loadContestConfig(configPath);
     if (config != null && config.validate() && setUpLogging(config)) {
+      try {
+        Logger.log(Level.INFO, "Begin config file contents:");
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: Is this the thing that got accidentally deleted in a merge? I swear you added this already.

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 3]
```diff
@@ -105,11 +107,28 @@ void tabulate() throws TabulationCancelledException {
...
+        Logger.log(Level.SEVERE, "Error logging config file: %s\n", configPath, e.toString());
+      }
+      Logger.log(Level.INFO, "Tabulating \'%s\'...", config.getContestName());
       if (config.isSequentialMultiSeatEnabled()) {
+        Logger.log(Level.INFO, "This is a sequential multi-seat contest.");
         int numWinners = config.getNumberOfWinners();
         // temporarily set config to single-seat so we can run sequential elections
         config.setNumberOfWinners(1);
         while (config.getSequentialWinners().size() < numWinners) {
+          Logger.log(Level.INFO, "Beginning sequence: %d...",
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: "Beginning tabulation for seat #%d..." would be clearer for me.

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 4]
```diff
@@ -121,6 +140,10 @@ void tabulate() throws TabulationCancelledException {
           String newWinner = (String) newWinnerSet.toArray()[0];
           config.setCandidateExclusionStatus(newWinner, true);
           config.addSequentialWinner(newWinner);
+          Logger.log(Level.INFO, "Sequence: %d complete.", config.getSequentialWinners().size());
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: "Tabulation for seat #%d complete."

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 5]
```diff
@@ -121,6 +140,10 @@ void tabulate() throws TabulationCancelledException {
...
+          if (config.getSequentialWinners().size() < numWinners) {
+            Logger.log(Level.INFO, "Excluding %s from the next sequence.", newWinner);
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: "Excluding %s from the remaining tabulations."

File: src/main/java/network/brightspots/rcv/TabulatorSession.java [hunk 6]
```diff
@@ -137,6 +160,7 @@ void tabulate() throws TabulationCancelledException {
           runTabulationForConfig(config, castVoteRecords);
         }
       }
+      Logger.log(Level.INFO, "Tabulation session complete.  Results written to: %s", outputPath);
```
Comment (tarheel) [inline src/main/java/network/brightspots/rcv/TabulatorSession.java]: double space -> single

File: src/main/java/network/brightspots/rcv/TabulatorSession.java (hunk 2 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Yeah it's nutty.  This commit: "Merge pull request #314 from BrightSpots/bad_cvr_checks" a79cdd3371d94d7cd5af32d746420cfd052c810c done by the github web tool.  Both parent commits have this code in them so i don't understand why the merge did this.  I went through all the other changes and they check out.

File: src/main/java/network/brightspots/rcv/TabulatorSession.java (hunk 3 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Yeah that's good idea

File: src/main/java/network/brightspots/rcv/TabulatorSession.java (hunk 6 above)
Comment (moldover) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Why this?

Comment (HEdingfield) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: The double space after a period is a Satanic Practice.

Comment (tarheel) [reply src/main/java/network/brightspots/rcv/TabulatorSession.java]: Yeah, double-spacing after a period is a relic of a bygone era. In any case, we use single spaces everywhere else.

**Pull request merged**